src/main/twirl/uk/gov/hmrc/play/views/helpers/dateFieldsFreeInline.scala.html

```diff
@@ -0,0 +1,58 @@
+@*
+* Copyright 2016 HM Revenue & Customs
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*   http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*@
+@(formItem:Form[_], fieldName: String, args: (Symbol,Any)*)(implicit lang: play.api.i18n.Lang)
+
+@import uk.gov.hmrc.play.mappers.DateFields._
+@import play.api.i18n._
+@import uk.gov.hmrc.play.mappers.DateFormatSymbols._
+
+
+
+@fieldsetClasses = @{if(formItem(fieldName).hasErrors) "form-field--error" else "" }
+
+<fieldset class="form-group form-date @fieldsetClasses" id="@fieldName-fieldset">
+
+    @formItem(fieldName).errors.map { error => <span class="error-notification">@Messages(error.message)</span>}
+
+    @input(
+        formItem(s"${fieldName}.day"),
+        '_label -> Messages("date.fields.day"),
+        '_labelClass -> "form-group form-group-day label--inlineTextinput form-field--inline-text-input",
```

Comment:
`form-field--inline-text-input` also isn't needed for outputting this. It'll also avoid the need for `margin: 0` in https://github.com/hmrc/assets-frontend/pull/534